app/views/admin/tasks/_form.html.erb

```diff
@@ -1,12 +1,29 @@
+<% translation_base = "#{claim.policy.to_s.underscore}.admin.task_questions.#{task_name}" %>
 <%= form_with model: [:admin, @claim, @claim.tasks.where(name: task_name).first_or_initialize] do |f| %>
   <div class="govuk-form-group">
     <fieldset class="govuk-fieldset govuk-!-margin-bottom-6">
       <legend class="govuk-fieldset__legend govuk-fieldset__legend--l ">
         <h3 class="govuk-heading-l">
-          <%= I18n.t("#{claim.policy.to_s.underscore}.admin.task_questions.#{task_name}") %>
+          <%= I18n.t("#{translation_base}.title", claim_full_name: @claim.full_name) %>
         </h3>
       </legend>
 
+      <%
+          I18n.t(
+            "#{claim.policy.to_s.underscore}.admin.task_questions.#{task_name}.body",
```

Comment:
@nomasprime Could this not use the translation_base variable that is on line 1?

eg
```
I18n.t("#{translation_base}.body"
```

Reply:
😆 Pretty much the reason I did it, must have been distracted 👍